### PR TITLE
EZP-28195: Cannot open Roles list because of Internal Server Error

### PR DESCRIPTION
**Pull request by Nattfarinn**
JIRA: https://jira.ez.no/browse/EZP-28195

### CODE REVIEW

File: src/lib/Form/Factory/FormFactory.php [hunk 1]
```diff
@@ -525,8 +525,9 @@ public function deleteRoleAssignment(
         ?string $name = null
     ): FormInterface {
         $name = $name ?: sprintf('delete-role-assignment-%s', md5(implode('/', [
-            $data->getRoleAssignment()->getRole()->id,
-            $data->getRoleAssignment()->getRoleLimitation()->getIdentifier(),
+            $data->getRoleAssignment()->getRole()->id, !empty($data->getRoleAssignment()->getRoleLimitation())
```
Comment (bdunogier) [inline src/lib/Form/Factory/FormFactory.php]: It's very personal, but I find that complicated to read when going through the code. A ternary method call inside an md5 of an implode of... :)

Comment (Nattfarinn) [reply src/lib/Form/Factory/FormFactory.php]: Because you're not so good in reading kung-fu!

**Comment by Nattfarinn** [comment]
@bdunogier Fixed :)

File: src/lib/Form/Factory/FormFactory.php (hunk 1 above)
Comment (bdunogier) [reply src/lib/Form/Factory/FormFactory.php]: You know how it goes: maintaining code is twice as hard than initially implementing it. If you're at 100% when writing code, you need to be at 200% when maintaining it.

Conclusion: keep it simple :)

**Pull request merged**